std/typecons.d

```diff
@@ -51,12 +51,12 @@ Encapsulates unique ownership of a resource.  Resource of type $(D T) is
 deleted at the end of the scope, unless it is transferred.  The
 transfer can be explicit, by calling $(D release), or implicit, when
 returning Unique from a function. The resource can be a polymorphic
-class object, in which case Unique behaves polymorphically too.
+class object or instance of an interface, in which case Unique behaves polymorphically too.
 */
 struct Unique(T)
 {
 /** Represents a reference to $(D T). Resolves to $(D T*) if $(D T) is a value type. */
-static if (is(T:Object))
+static if (is(T == class) || is(T == interface))
```

Comment:
I don't know about `extern(Windows)` - isn't that similar to `extern(C)` which wouldn't work with classes (I really haven't done much with Windows and D, and Windows linking is way more complicated in general than *nix)? However, as for something like `extern(C++)` or COM interfaces, wouldn't checking

    is(T : Object) && (is(T == class) || is(T == interface))

do the trick?